### PR TITLE
Add a warning about routes parameters names limitations

### PR DESCRIPTION
**Pull request by fancyweb**

### CODE REVIEW

File: routing.rst [hunk 1]
```diff
@@ -466,6 +466,10 @@ a slash. URLs matching this route might look like:
     Symfony provides you with a way to do this by leveraging service container
     parameters. Read more about this in ":doc:`/routing/service_container_parameters`".
 
+.. caution::
+
+  A route parameter name cannot start with a digit and cannot be longer than 32 characters.
```
Comment (wouterj) [inline routing.rst]: Please use 4 spaces to indent the content of the caution directive.

**Comment by wouterj** [comment]
👍 (left one minor comment, but this can be fixed while merging)

status: reviewed

**Comment by xabbuh** [comment]
👍

**Comment by xabbuh** [comment]
Hm, what do you think to move this up to the "Routing Examples" section to make it more visible to the reader?

**Pull request closed without merging**